src/app/pages/IdxPage/idxPageDataPropTypes.js

```diff
@@ -0,0 +1,12 @@
+import { arrayOf, shape } from 'prop-types';
```

Comment:
I wonder if we should also add this file under `models`, as we do with the front page.